backend/src/global/types/dataEditTypes.ts

```diff
@@ -22,3 +22,5 @@ export type OfferCreate = Omit<
 };
 
 export type InvoiceUpdate = Partial<Omit<Invoice, "id">>;
+
+export type OfferUpdate = Partial<Pick<Offer, "status">>;
```

Comment:
Are you sure you want to use "Pick"?

This means the status is the only thing you can update for the offer. Is that the case?